src/main/java/io/github/thebusybiscuit/sensibletoolbox/util/STBUtil.java

```diff
@@ -723,4 +729,36 @@ public static boolean isPotionIngredient(Material type) {
         // TODO Fix Potion Ingredient lookup
         return false;
     }
+
+    /**
+     * Get the wall sign version of the sign material e.g. OAK_SIGN -> OAK_WALL_SIGN
+     *
+     * @param signType
+     *                 The material of the sign
+     * @return The wall sign version of that sign
+     */
+    @Nullable
+    public static Material getWallSign(@Nonnull Material signType) {
+        Validate.notNull(signType);
```

Comment:
Please add a message to the validation
```suggestion
        Validate.notNull(signType, "The Sign Type cannot be null");
```